Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -985,10 +985,7 @@ public string SelectedText
         {
             get
             {
-                var text = selectedText ?? selectedTextParam;
-                return string.IsNullOrEmpty( text )
-                    ? ( FreeTyping ? CurrentSearch : ( text ?? string.Empty ) )
-                    : ( text ?? string.Empty );
```

Comment:
Is this changing the logic of an actual implemented working feature? So the end user will have it working differently?  
Any of this changes should be documented on the release notes. Also I don't agree with it.